src/index.tsx

```diff
@@ -248,10 +252,12 @@ const ReactTagsWrapper = (props: ReactTagsWrapperProps) => {
     inputValue,
     maxTags,
     renderSuggestion,
+    labelledById,
```

Comment:
@shrilakshmishastry why this needs to be a user prop ? We can hardcode it instead ?

Ww can append a random id to it so that if there are multiple tags, every tag component has a unique id. We can generate the id using [crypto.randomUUID](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) and shorten its length

```
const id = crypto.randomUUID().slice(0, 8)
```

Reply:
Please explain a bit more about hardcoding part.

Context of labelledById here : 
This is the title or label given to describe the context in which tag is being used. In the example set up we have, it's `react tags example`. 

To describe how a screen reader user feels when he goes through the tags without the label being add is he reads heading `react tags example` then read tags lets say India is first one, it will be read as `India`. India can be taken as description also .

Reply:
After reading the second sentence in your comment, I felt that you're considering labeling the tag when it's not been edited. 

I am adding the label to tag when it is being edited 😄

Reply:
I meant hardcoding the label as `React Tags`. lets do this, have a default value `React Tags`, it shouldn't be a required prop, we should have always have a default/fallback value.
Additonally I realized the prev json structure we decided isn't the best so lets improve that and instead of `ariaAttrs` based on the component.

so for clear all button it would be 

```
clearAllBtnAriaAttrs?: {
    aria-label?: string
}
```

And for input it will be `inputAria Attrs`

```
inputAriaAttrs?: {
    aria-labelledby?:string
 }
```

This structure looks more clean and has separation for components. Wdyt?

Reply:
> I agree with the prop-type structure. But I am not ok with the label being React tags. Let's say I have tags of countries, how come React tags make sense here?

That would be the default label, we can simplify it to `tags` or some other name if you have better suggestions :)
We need a default generic label that would be used in case the users don't pass the label explicitly

Reply:
re-structured the props now to accept accessibility attributes based on component

https://github.com/react-tags/react-tags/blob/7fbd38bde1a6b3be2e592b6516e16a330fa05f9e/src/index.tsx#L205

https://github.com/react-tags/react-tags/blob/7fbd38bde1a6b3be2e592b6516e16a330fa05f9e/src/index.tsx#L214

Made the default label as tags https://github.com/react-tags/react-tags/blob/7fbd38bde1a6b3be2e592b6516e16a330fa05f9e/src/components/ReactTags.tsx#L530